runtime/kusama/src/lib.rs

```diff
@@ -354,6 +357,11 @@ parameter_types! {
 
 	// miner configs
 	pub OffchainRepeat: BlockNumber = 5;
+
+	/// Whilst `UseNominatorsAndUpdateBagsList` or `UseNominatorsMap` is in use, this can still be a
+	/// very large value. Once the `BagsList` is in full motion, staking might open its door to many
+	/// more nominators, and this value should instead be what is a "safe" number (e.g. 22500).
+	pub const VoterSnapshotPerBlock: u32 = u32::MAX;
```

Comment:
@kianenigma why not put a safe number to start with?

Sounds easy to forget about this.

Reply:
I @emostov recommended the same, sure it can already be 22500